nltk/corpus/reader/verbnet.py

```diff
@@ -6,10 +6,10 @@
 # For license information, see LICENSE.TXT
 
 """
-An NLTK interface to the VerbNet verb lexicon
+An NLTK interface to the Verbnet verb lexicon
 
-For details about VerbNet see:
-http://verbs.colorado.edu/~mpalmer/projects/verbnet.html
+For details about Verbnet see:
+http://verbs.colorado.edu/~mpalmer/projects/Verbnet.html
```

Comment:
it's http://verbs.colorado.edu/~mpalmer/projects/verbnet.html 

Verbnet.html doesn't work.

Also, it might worth leaving things as they were, that is to use 'VerbNet', not 'Verbnet'.